SRC/NETWORK/hget.c

```diff
@@ -123,32 +135,32 @@ enum TcpipErrorCodes {
 #define strDefaultFilename "index.htm";
 
 const char* strTitle=
-    "HTTP file downloader 1.1\r\n"
-    "By Konamiman, 1/2011\r\n"
+    "HTTP file downloader 1.3\r\n"
+    "By Konamiman, 2011 v1.3 Oduvaldo (ducasp@gmail.com) 07/2019\r\n"

```

Comment:
Alternative wording that imho will read better:

By Oduvaldo (ducasp@gmail.com), 7/2019
Based oh HGET 1.1 by Konamiman

Reply:
I've done it, but on a single line, otherwise when printing instructions it would scroll down and not show the application name/version.